src/components/StoreInfo/StoreInfo.tsx

```diff
@@ -1,46 +1,41 @@
 import * as React from 'react';
 import styles from './styles.module.scss';
-import StoreDetails, { StoreDetailsProps } from '../StoreDetails';
+import StoreDetails from '../StoreDetails';
 import classnames from 'classnames';
+import { Address, Seller } from '../../utilities';
 
-interface Props {
-  nationality: string;
-  name: string;
-  address: {
-    line1: string;
-    line2: string;
-  };
-  phoneNumber: string;
-  introduction: string;
-  className?: string;
-  storeDetailsProps: StoreDetailsProps;
-}
+type Props = {
+  seller: Seller;
+};
+
+export const StoreInfo: React.SFC<Props> = ({ seller }) => {
+  const {
+    summary,
+    story,
+    className,
+    cuisineName,
+    addresses,
+    phoneNumber,
+  } = seller;
 
-const StoreInfo: React.SFC<Props> = ({
-  nationality,
-  name,
-  address,
-  phoneNumber,
-  introduction,
-  className,
-  storeDetailsProps,
-}) => {
-  const detailsProps = {
-    ...storeDetailsProps,
-    className: styles.storeDetails,
-  };
   return (
     <section className={classnames(styles.container, className)}>
-      <div className={styles.nationality}>{nationality}</div>
+      <div className={styles.nationality}>{cuisineName}</div>
       <p>
-        <div className={styles.address}>{address.line1}</div>
-        <div className={styles.address}>{address.line2}</div>
+        {addresses &&
```

Comment:
@kevinjtsai I added support of multiple addresses in the case that they have them. Could we maybe see how design would like for this to look?